### ISSUE TITLE
autoconverter uuid

### ISSUE THREAD

**Issue by Diclo** [games647/FastLogin#268]
greeting
could it be done that when activating the / premium
 the save.dat and the playerdata of essentials were converted automatically?

**Comment by LoneDev6** [comment]
I don't think this plugin is doing that automatically, this can be considered as a feature request

**Comment by 7wOv6ySCjo** [comment]
Clearly, this would be a good feature.

**Comment by Govindass** [comment]
for playeruuid.dat files I think this would be good, but for plugin specific ones like Essentials, that's overkill because you would need to add support for every plugin in existence and that differs in every plugin, but for start, it could be nice to specify list of directories for where to look for yaml files/plain text to replace old uuid with new, this one could work automated with just configuration as long as the plugins use yaml/plain text to save data, but then the problem could come to unexpected stuff, such as 2 plugins editing same file at once, etc. this is best done manually and not be feature of FastLogin I think
